src/utils/exemplars.ts

```diff
@@ -0,0 +1,42 @@
+import { map, Observable } from 'rxjs';
+import { DataFrame, DataTopic, Field } from '@grafana/data';
+import { CustomTransformerDefinition, SceneObject } from '@grafana/scenes';
+import { getTraceExplorationScene } from './utils';
+
+export const exemplarsTransformations = (scene: SceneObject): CustomTransformerDefinition[] => [
+  {
+    topic: DataTopic.Annotations,
+    operator: () => (source: Observable<DataFrame[]>) => {
+      const traceExplorationScene = getTraceExplorationScene(scene);
+      return source.pipe(
+        map((data: DataFrame[]) => {
+          return data.map((frame) => {
+            if (frame.name === 'exemplar') {
+              const traceIDField = frame.fields.find((field: Field) => field.name === 'traceId');
+              if (traceIDField) {
+                // The traceID will be interpolated in the url
+                // Then, onClick we retrieve the traceId from the url and navigate to the trace exploration scene by setting the state
+                traceIDField.config.links = [
+                  {
+                    title: 'View trace',
+                    url: '#${__value.raw}',
+                    onClick: (event) => {
+                      event.e.stopPropagation(); // Prevent the click event from propagating to the parent anchor
+                      const parentAnchorHref = event.e.target.parentElement.parentElement.href;
+                      const traceId = parentAnchorHref.split('#')[1];
+                      traceExplorationScene.state.locationService.partial({
+                        traceId,
+                      });
```

Comment:
Can we be a bit more careful here please. Events/parentElement/targets are notorious for being undefined or unexpected. A few optional chaining ?'s would be safer so we don't get any potential runtime errors, especially if changes to the DOM/parent structure occur. Same with the `split('#')[1]` in case there are not even two parts to access the second.

We're also assuming `parentAnchorHref.split('#')[1]` will always be the traceId, but we don't check the formatting to be sure it's a traceId. We don't want to update the state with an invalid traceid!

Adding a fallback so that we link to Classic Explore if any of the above is a problem would be ideal but not required.